docs/cli.md

```diff
@@ -121,7 +121,7 @@ Use `--instance` and `--parameter`/`-p` for setting an Instance name and Paramet
 $ kubectl kudo install kafka --instance=my-kafka-name --parameter="KAFKA_ZOOKEEPER_URI,zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181" --parameter="KAFKA_ZOOKEEPER_PATH,/small" -p="BROKERS_COUNTER,3"
```

Comment:
Let's pick up from where we left off ( https://github.com/kudobuilder/kudo/pull/283#pullrequestreview-245024206 ) and also fix/update to the now working command: `kubectl kudo install kafka --instance=my-kafka-name --parameter KAFKA_ZOOKEEPER_URI=zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181 --parameter KAFKA_ZOOKEEPER_PATH=/small -p BROKERS_COUNTER=3`

Reply:
Good catch, done! =)